web3/main.py

```diff
@@ -343,6 +344,12 @@ def pm(self) -> "PM":
             )
 
     def enable_unstable_package_management_api(self) -> None:
+        warnings.warn(
+            "The ``ethPM`` module is no longer being maintained and will be "
+            "deprecated with ``web3.py`` version 7",
+            UserWarning,
```

Comment:
I'm thinking a `DeprecationWarning` might make more sense here?

Reply:
I had initially tried that, but it didn't show the warning when I imported it. Either way :shrug:

Reply:
`DeprecationWarning` is usually turned off by default. You have to want to see them. I just turned it on in a script and ran it with the change and it shows up nicely. I do lean towards using that since it seems more appropriate in this case.

Reply:
@pacrob lgtm 👍🏼